### PR TITLE
cloud-dev: Add some more common dependencies

### PR DESCRIPTION
**Pull request by jyn514**
Most of these are self-explanatory. Powershell is useful for testing changes to `./x.ps1`. `delta` is a program for making `git diff` easier to read.

### CODE REVIEW

File: ansible/roles/dev-desktop/tasks/main.yml [hunk 1]
```diff
@@ -209,8 +218,18 @@
...
+      - jq
+      - powershell
     state: present
 
+- name: Install git-delta
+  shell: |
+    DELTA_VERSION=$(curl --silent https://api.github.com/repos/dandavison/delta/releases/latest | jq --raw-output '.name')
+    curl -O git-delta.deb https://github.com/dandavison/delta/releases/download/$DELTA_VERSION/git-delta_$DELTA_VERSION_$(uname -m).deb
+    apt install ./git-delta.deb
+    rm ./git-delta.deb
```
Comment (jyn514) [inline ansible/roles/dev-desktop/tasks/main.yml:231]: admittedly this is kinda janky; I'm ok with installing it just for my user if you don't think other people on the machine will use it.

**Review by jdno — COMMENTED** [review]
I suggest that we split the Powershell and git-delta into their own PRs, and get this merged to deploy the other tools asap.

File: ansible/roles/dev-desktop/tasks/main.yml (hunk 1 above)
Comment (jyn514) [reply ansible/roles/dev-desktop/tasks/main.yml]: 👍 makes sense, I don't think a lot of other people are using it, happy to remove it

**Comment by jyn514** [comment]
Ok, pushed a change removing powershell and `git-delta` :)

**Review by jdno — APPROVED** [review]

**Pull request merged**